nmn.Templates/src/templateController.js

```diff
@@ -26,6 +27,8 @@ addTag('matchingEvents', listMatchingDaysEvents)
 addTag('listMatchingEvents', listMatchingDaysEvents)
 addTag('quote', getDailyQuote, true)
 addTag('sweepTasks', sweepTemplate)
+addTag('reminders', listReminders)
```

Comment:
I would prefer a single tag for this. At least until we can add some sort of forEach syntax.